css/_sidenavs.scss

```diff
@@ -110,3 +110,14 @@
     margin-right: 10px!important;
     margin-left: 5px!important;
 }
+
+md-sidenav {
```

Comment:
Instead of relying on tags, it could be better to use some css classes (cf [MDN guide](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Writing_efficient_CSS#Try_to_put_rules_into_the_most_specific_category_you_can.21)).
Is it possible in that case? Thanks